features/support/tag_helpers.rb

```diff
@@ -1,3 +1,28 @@
+# Creates a tag with the given name in the local repo
+def create_local_tag tag_name
+  p "CREATING LOCAL TAG"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.